airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -350,6 +355,11 @@ protected boolean isNotInternalSchema(final JsonNode jsonNode, final Set<String>
   // state
   @Override
   protected List<AirbyteStateMessage> generateEmptyInitialState(final JsonNode config) {
+    if (!featureFlags.useStreamCapableState()) {
+      return List.of(new AirbyteStateMessage()
+          .withType(AirbyteStateType.LEGACY)
+          .withData(Jsons.emptyObject()));
+    }
```

Comment:
@benmoriceau it looks like the non-overriden method does `.withData(Jsons.jsonNode(new DbState()))`, but here you are using `.withData(Jsons.emptyObject())`: https://github.com/airbytehq/airbyte/blob/7bff12aea514bfbce1545d060fde2311f764503c/airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/AbstractDbSource.java#L551
Should these be the same?

Reply:
I think so. I have updated it